cmd/buildah/common.go

```diff
@@ -71,12 +72,20 @@ func getStore(c *cobra.Command) (storage.Store, error) {
 		}
 		uopts := globalFlagResults.UserNSUID
 		gopts := globalFlagResults.UserNSGID
+
```

Comment:
In Podman if you specify one without the other, then it is assumed you want the same for both.

--uidmap 0:10000:100
Implies
--gidmap 0:10000:100

I think we should be consistent in Buildah.